prepare_offline_installation/prepare_offline_installation.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# simplify your orcharhino Server offline installation by generating commands based on a list of repositories
+
+###################################################################
+# adapt path to your offline medium, for example an external disk
+repo_path = "/media/my_disk/my_repos/"
```

Comment:
might be nicer to have this as a command-line parameter; if it is the only one it would be fine to always use the first parameter `$1`

Reply:
Applied!